build.yaml

```diff
@@ -772,7 +772,9 @@ run_trial_modules: &run_trial_modules
   - flocker.dockerplugin
   - flocker.node.agents
   - flocker.node.test
-  - flocker.node.functional.test_docker
+  - flocker.node.functional.test_docker.GenericDockerClientTests
+  - flocker.node.functional.test_docker.IDockerClientNamespacedTests
+  - flocker.node.functional.test_docker.NamespacedDockerClientTests
   - flocker.node.functional.test_script
```

Comment:
FWIW, the `@flaky` decorator is not yet activated for functional tests, so whatever slowdown we're seeing is unrelated.